### PR TITLE
Add ansible.windows as a galaxy dependency

### PR DESCRIPTION
**Pull request by jcheng-splunk**

### CODE REVIEW

File: deployments/ansible/molecule/default/Dockerfile.j2 [hunk 1]
```diff
@@ -1,9 +1,5 @@
 {% if item.image not in ["opensuse12", "opensuse15"] %}
 FROM geerlingguy/docker-{{ item.image }}-ansible:latest
-{% if item.image == "centos8" %}
-RUN sed -i 's|\$releasever|8-stream|g' /etc/yum.repos.d/CentOS*.repo
-RUN dnf install -y --allowerasing centos-stream-release
-{% endif %}
 {% else %}
```
Comment (dloucasfx) [inline deployments/ansible/molecule/default/Dockerfile.j2:3]: Was this change on purpose?

**Review by dloucasfx — CHANGES_REQUESTED** [review]

File: deployments/ansible/molecule/default/Dockerfile.j2 (hunk 1 above)
Comment (jcheng-splunk) [reply deployments/ansible/molecule/default/Dockerfile.j2:3]: It was a workaround until the [centos8 image](https://hub.docker.com/r/geerlingguy/docker-centos8-ansible) was updated for the new `8-stream` repo.  Otherwise the docker build would fail.
